src/Message/PurchaseRequest.php

```diff
@@ -62,13 +62,41 @@ protected function getFunction()
         return 'authCapture';
     }
 
-    public function getData($paymentMethodType = self::PAYMENT_METHOD_CREDIT_CARD)
+    public function getData(string $paymentMethodType = self::PAYMENT_METHOD_CREDIT_CARD)
     {
         $data = parent::getData($paymentMethodType);
 
-        $data['ignoreAvsPolicy'] = false;
-        $data['ignoreCvnPolicy'] = false;
+        $ignore_avs = $this->getIgnoreAvsPolicy();
```

Comment:
Could you update the doc block above to include this param, now that it isn't set by default? And a short comment about what this is for?